netman/adapters/switches/brocade.py

```diff
@@ -439,10 +439,12 @@ def parse_vlan(vlan_data):
     for line in vlan_data[1:]:
         if regex.match("^\srouter-interface ve (\d+)", line):
             current_vlan.vlan_interface_name = regex[0]
-        elif regex.match(" tagged (.*)", line):
-            for name in parse_if_ranges(regex[0]):
-                current_vlan.tagged_interfaces.append(name)
-
+        elif regex.match("^\s(tagged|no untagged)\s(.*)$", line):
+            type = regex[0]
+            for name in parse_if_ranges(regex[1]):
+                current_vlan.interfaces.append(name)
+                if type == "tagged":
+                    current_vlan.tagged_interfaces.append(name)
```

Comment:
This field does not exist on the vlan object, and why do we want 2 separate lists?   and if we do have 2 separate lists, why isn't the first one called untagged?

We need this information to identify on which interfaces one vlan is tagged/untagged, right now netman uses cisco's naming standard:
untagged = access-vlan or native-vlan
tagged = trunk vlan

BUT... it came to my mind that this naming convention tagged/untagged could abstract the concept of access-vlan vs native-vlan caus in the end they are the same thing....

interesting discussion